metricbeat/metricbeat.full.yml

```diff
@@ -90,6 +90,38 @@ metricbeat.modules:
   # Password of hosts. Empty by default
   #password: test123
 
+#-------------------------------- ceph Module --------------------------------
+- module: ceph
+  metricsets: ["perf"]
+  enabled: true
+  period: 1s
+
+ceph:  
```

Comment:
This part of the config must be moved under the module. I assume it doesn't work the way it is at the moment.